core/ui/PageControls/reset-layout.tid

```diff
@@ -0,0 +1,15 @@
+title: $:/core/ui/Buttons/reset-layout
+tags: $:/tags/PageControls $:/tags/LayoutSwitcherButton
+caption: {{$:/core/images/reset-layout-button}} {{$:/language/Buttons/ResetLayout/Caption}}
+description: {{$:/language/Buttons/ResetLayout/Hint}}
+
+\whitespace trim
```

Comment:
Since there is `\whitespace trim` the code itself should use indentation. .. We use tabs to indent code.

Reply:
Ok, Seems I was using 2 space in some of previous code...